.golangci.yml

```diff
@@ -0,0 +1,89 @@
+version: "2"
+linters:
+  enable:
+    # Commented entries here are not enabled by default, but that we should
+    # add once the relevant lines are fixed up.
+    
+    # - errorlint # found 2 cases to be fixed.
+    # - exhaustive # found some cases that should be fixed.
+    # - exhaustruct # found some structs which missed fields in initializing.
+    # - inamedparam  # fix missing named param
+    # - makezero  # fix the unallocated elements.
+    # - misspell  # fix spelling
+    # - nlreturn  # fix these missing blank line
+    # - nonamedreturns  # found some shadowing
+    # - predeclared  # found some shadowing
+    # - revive  # detects a number of good fixes
+    # - thelper  # fix test methods with t.Testing not as first param
+    - unconvert
+    # - wastedassign  # fix and enable
+    # - whitespacew  # fix and enable
+  disable:
+    # A number of these checks run by default so they are explicitly
+    # disabled here. 
+    #
+    # Other of these checks are disabled by default, but we want to
+    # comment on why we plan to keep them excluded going forward.
+    #
+    # TODO(rsned): Move the checks to enable: once the flagged 
+    # problems are resolved for each check.
+    # https://github.com/golang/geo/issues/145
+   
+    # Enable one we reduce the cleverness with unicode chars.
```

Comment:
Enable *once* (same error on line 39).  Let me know if you'd prefer me to fix simple obvious stuff like this for you.  It feels weird to make commits to your fork where you don't have a chance to review them.

Reply:
Sure, simple stuff like that is fine.